ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,52 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+### Segment Routing Summary
+
+- SRTE is enabled.
+
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+- system-colored-tunnel-rib is enabled
+{%         endif %}
+
+### SRTE Policies
+
+| Endpoint | Color | Preference | Name | Description | Label Stack | Index  | Weight | Explicit Null |
+| -------- | ----- | ---------- | ---- | ----------- | ----------- | ------ | ------ | ------------- |
+{%         if router_traffic_engineering.segment_routing.policy_endpoints is arista.avd.defined %}
+{%             for endpoint in router_traffic_engineering.segment_routing.policy_endpoints | arista.avd.natural_sort %}
+{%                 for color in endpoint.colors | arista.avd.natural_sort %}
+{%                     set name = color.name | arista.avd.default('-') %}
+{%                     set description = color.description | arista.avd.default('-') %}
+{%                     for pathgroup in color.path_group | arista.avd.natural_sort %}
+{%                         set pathgroup_preference = pathgroup.preference | arista.avd.default('-') %}
+{%                         if pathgroup.explicit_null_ipv4 is arista.avd.defined(true) %}
+{%                             set expnull = "ipv4" %}
+{%                         endif %}
+{%                         if pathgroup.explicit_null_ipv6 is arista.avd.defined(true) %}
+{%                             set expnull = "ipv6" %}
+{%                         endif %}
+{%                         if pathgroup.explicit_null_ipv4 is arista.avd.defined(true) and pathgroup.explicit_null_ipv6 is arista.avd.defined(true) %}
+{%                             set expnull = "ipv4 ipv6" %}
+{%                         endif %}
+{%                         set expnull = expnull | arista.avd.default('-') %}
+{%                         for labelstack in pathgroup.segment_list | arista.avd.natural_sort %}
+{%                             set stack = labelstack.label_stack %}
+{%                             set index = labelstack.index | arista.avd.default('-') %}
+{%                             set weight = labelstack.weight | arista.avd.default('-') %}
+| {{ endpoint.address }} | {{ color.value }} | {{ pathgroup_preference }} | {{ name }} | {{ description }} | {{ stack }} | {{ index }} | {{ weight }} | {{ expnull }} |
+{%                         endfor %}
+{%                     endfor %}
+{%                 endfor %}
+{%             endfor %}
+{%         endif %}
+
+### Segment Routing Traffic Engineering Device Configuration
```

Comment:
Line 46-50 should be out of endif in line 51. Because this is config template for Traffic Engineering, we don't want this to be inside segment-routing if condition.